pulp_2to3_migration/app/plugin/docker/utils.py

```diff
@@ -0,0 +1,21 @@
+from . import pulp2_models
+
+from pulp_2to3_migration.pulp2 import connection
+
+
+connection.initialize()
+
+
+def find_tags():
+    """
+    Find tags that have same name within the repo.
+    Return only one tag out of 2 tags with the same name.
```

Comment:
To make more connection to the aggregation pipeline code, maybe you can say something like :
`Return only the first tag (== schema 2 tag) out of 2 tags with the same name`.
Maybe it's not a good idea. I just worry that if I read this in a month from now, it would take me a while to understand the `group_stage1` and the `$first` thingy.
Up to you :)